src/components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
```

Comment:
In the future, better give names for props like NotificationsProps.

Reply:
I like to use Props in every component it feels more structurized for me.

Reply:
Agree with @roshirin, just use Props everywhere